maintainers/maintainer-list.nix

```diff
@@ -6303,6 +6303,12 @@
     githubId = 1843676;
     name = "Jonathan Reeve";
   };
+  jonnybolton = {
+    email = "jonnybolton@gmail.com";
+    github = "jonnybolton";
+    githubId = 8580434;
+    name = "Jonny Bolton";
+  };
```

Comment:
Different commit: `maintainers: add jonnybolton`

Reply:
I have moved the change to maintainer-list.nix into #177681

Reply:
Don't do a _different pull request_, but a _different commit_.

Reply:
Apologies, I misunderstood. This should be correct now.